### PR TITLE
Add Team page

### PR DESCRIPTION
**Pull request by AbdBarho**
Basic skeleton of the team page, will be improved upon later.

I did not add all people because that would take ages, I will write in discord after the PR is merged for the people to add themselves to the json.

image

Refs #1703

### CODE REVIEW

File: website/src/pages/team.tsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+                  <Text as="h2" fontWeight="bold" size="lg">
+                    {group.name}
+                  </Text>
+                  <Grid gap={6} gridTemplateColumns="repeat(auto-fit, minmax(300px, 1fr))">
+                    {group.members.map((id) => {
+                      const info = people[id] ?? {};
+                      const { name, title, githubURL, imageURL } = info;
+                      return (
+                        <Flex key={id} gap={2}>
+                          <Avatar src={imageURL} />
```
Comment (notmd) [inline website/src/pages/team.tsx:38]:
```suggestion
                          <Avatar src={imageURL} loading="lazy"/>
```

**Review by notmd — APPROVED** [review]
Nice. Have you considered loading other contributors using Github API?
Or we can use this instead https://contrib.rocks/preview?repo=angular%2Fangular-ja

**Comment by AbdBarho** [comment]
> Nice. Have you considered loading other contributors using Github API? Or we can use this instead https://contrib.rocks/preview?repo=angular%2Fangular-ja

Yeah, this was discussed in #1703, the problem is that we have contributors who were never active on github, like moderators / legal team and stuff, and frankensteining a solution that works for both would be annoying.

**Review by yk — APPROVED** [review]
thank you!

**Comment by yk** [comment]
merged this so we can inform people about the page

**Pull request merged**